crates/router/src/compatibility/stripe/setup_intents/types.rs

```diff
@@ -313,15 +313,17 @@ pub enum StripeSetupStatus {
 impl From<api_enums::IntentStatus> for StripeSetupStatus {
     fn from(item: api_enums::IntentStatus) -> Self {
         match item {
-            api_enums::IntentStatus::Succeeded => Self::Succeeded,
+            api_enums::IntentStatus::Succeeded | api_enums::IntentStatus::PartiallyCaptured => {
```

Comment:
Isn't it confusing? We're basically mapping `PartiallyCaptured` as `Succeeded` along with a successful payment.

Reply:
Since Stripe considers partial captured payment as succeeded, I think is safe to map `PartiallyCaptured` to `Succeeded`

Reply:
Since Stripe considers partial captured payment as succeeded, I think is safe to map `PartiallyCaptured` to `Succeeded`